build-ci-wheels/action.yml

```
@@ -0,0 +1,82 @@
+name: "Build wheels for a C-extension Python library"
+description: "Build wheels for a C-extension Python library"
+
+inputs:
+  python-version:
+    description: 'Desired Python version'
+    default: '3.10'
+    required: false
+    type: string
+  cibw-build:
+    description: 'Desired build wheel CPython version'
+    default: "cp310-*"
+    required: false
+    type: string
+  cibw-skip:
+    description: 'Desired build wheel for skipping'
+    default: "*-musllinux* "
+    required: false
+    type: string
+  cibw-archs:
+    description: 'Desired build arhitecture'
+    default: "auto"
+    required: false
+    type: string
+  requires-pypy:
+      description: 'Whether requires build wheel for PyPy'
+      default: false
+      required: false
+      type: boolean
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Install Git and clone repository"
+      uses: actions/checkout@v3
+    
+    - name: Setup Python ${{ inputs.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ inputs.python-version }}
+   
+    - name: Set up QEMU
+      if: ${{ inputs.cibw-archs }} == 'aarch64'
```

Should apply GitHub syntax for string comparison:

```suggestion
      if: ${{ inputs.cibw-archs == 'aarch64' }}
```